package.json

```diff
@@ -112,12 +112,12 @@
     }
   },
   "lint-staged": {
-    "*.{js,ts}": [
+    "*.js": [
```

Comment:
When I originally made this PR, I forgot to test eslint on `ts` files, and it doesn't support `ts` right now, so I've moved them into the other hook.